.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is.
+
+## How to reproduce
+Steps to reproduce the behaviour:
+1. Go to '...'
+2. Click on '....'
```

Comment:
I know you probably just copied this, though for this package we're unlikely to ever have people clicking and scrolling so could change to be more generic, e.g.

> 1. '...'
> 2. '...'
> 3. '...'